docs/resources/Channel.md

```diff
@@ -609,7 +609,7 @@ Due to possible ambiguities, not all configurations are valid. An _invalid_ conf
 Because `parse: ["users"]` and `users: [123, 124]` are both present, we would throw a validation error.
 This is because the conditions cannot be fulfilled simultaneously (they are mutually exclusive).
 
-The ID list fields act as whitelists, and can contain mentions not in the content. These ID's that are not in the
+The ID list fields act as accepted IDs, and can contain mentions not in the content. These ID's that are not in the
 content will simply be ignored.
```

Comment:
```suggestion
Any entities with an ID included in the list of IDs will be mentioned. Note that IDs of entities not included in the content will have no effect.
```
Perhaps rephrase it to this?

Reply:
I'd honestly prefer to keep the original phrasing of the last part of the sentence. i.e.:
```suggestion
Any entities with an ID included in the list of IDs will be mentioned. Note that IDs of entities not included in the content will simply be ignored.
```
A bit more clear to me